<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>unkey</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextEdit" name="hash_textEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="src_textEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="rc_lineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>clicked(bool)</slot>
 </slots>
</ui>
